src/UsgsAstroLsSensorModel.cpp

```diff
@@ -420,10 +405,10 @@ void UsgsAstroLsSensorModel::updateState()
    // This routine will set some parameters not found in the ISD.
 
    // Reference point (image center)
-   double lineCtr = m_totalLines / 2.0;
-   double sampCtr = m_totalSamples / 2.0;
+   double lineCtr = m_nLines / 2.0;
+   double sampCtr = m_nSamples / 2.0;
    csm::ImageCoord ip(lineCtr, sampCtr);
-   double refHeight = m_refElevation;
+   double refHeight = 30;

```

Comment:
This should be changed to 0. It was 30 in reset, but was getting overwritten with 0 when the state was added. This is causing the attitude correction test to fail.